nipype/interfaces/niftyseg/maths.py

```diff
@@ -0,0 +1,427 @@
+# emacs: -*- mode: python; py-indent-offset: 4; indent-tabs-mode: nil -*-
+# vi: set ft=python sts=4 ts=4 sw=4 et:
+
+"""
+Nipype interface for seg_maths.
+
+The maths module provides higher-level interfaces to some of the operations
+that can be performed with the niftysegmaths (seg_maths) command-line program.
+
+Examples
+--------
+See the docstrings of the individual classes for examples.
+
+Change directory to provide relative paths for doctests
+    >>> import os
+    >>> filepath = os.path.dirname( os.path.realpath( __file__ ) )
+    >>> datadir = os.path.realpath(os.path.join(filepath, '../../testing/data'))
+    >>> os.chdir(datadir)
+"""
+
+import os
+
+from ..base import (TraitedSpec, File, traits, isdefined, CommandLineInputSpec,
+                    NipypeInterfaceError)
+from .base import NiftySegCommand, get_custom_path
+from ...utils.filemanip import split_filename
+
+
+class MathsInput(CommandLineInputSpec):
+    """Input Spec for seg_maths interfaces."""
+    in_file = File(position=2,
+                   argstr='%s',
+                   exists=True,
+                   mandatory=True,
+                   desc='image to operate on')
+
+    out_file = File(name_source=['in_file'],
+                    name_template='%s',
+                    position=-2,
+                    argstr='%s',
+                    desc='image to write')
+
+    _dtypes = ['float', 'char', 'int', 'short', 'double', 'input']
+
+    desc = 'datatype to use for output (default uses input type)'
+    output_datatype = traits.Enum(*_dtypes,
```

Comment:
This is totally stylistic, but _dtypes is not used anymore right?, why not having the options in the Enum initialization?